apps/site/eslint.config.js

```diff
@@ -35,6 +35,7 @@ export default tseslint.config(
       '@typescript-eslint/consistent-type-imports': 'error',
       '@typescript-eslint/no-require-imports': 'off',
       '@next/next/no-duplicate-head': 'off',
+      'import-x/no-duplicates': 'off',
```

Comment:
Is there a place where we are using a duplicate import? Shouldn't that be remediated instead of disabling the lint rule?

If it must be disabled, shouldn't it be for a specific file, rather than the entire `apps/site` project?

Reply:
Hmm, it is a personal bias on my side. I like to see clean, organized imports, and sometimes I rather split imports from the same module.